webui/react/src/components/ExperimentStopModal.tsx

```diff
@@ -24,6 +25,9 @@ interface Props {
 
 const ExperimentStopModalComponent: React.FC<Props> = ({ experimentId, onClose }: Props) => {
   const [type, setType] = useState<AvalableActions>(ActionType.Cancel);
+  const f_flat_runs = useFeature().isOn('flat_runs');
+
+  const buttonText = f_flat_runs ? 'Stop Search' : BUTTON_TEXT;
```

Comment:
nit: this feels like a footgun for a test that relies on `BUTTON_TEXT` failing

Reply:
there's nothing importing BUTTON_TEXT from this module, would it make sense to inline?

Reply:
Yeah, if BUTTON_TEXT isn't being used then definitely